scss/mixins/_remove-appearance.scss

```diff
@@ -0,0 +1,8 @@
+// remove initial styles from chrome & safari
+@mixin remove-appearance {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  border-radius: 0;
+  outline: none;
+}
```

Comment:
One more newline. Since it's only for `.scss` files I'm guessing they're being taken care of automatically for everything but them by your editor. Might want to poke around settings to avoid the hassle moving forward.

Reply:
I'm looking into this and it's sort of a mystery.  I have this lint setting in place, but it isn't working.  Seems like there was a bug in scss-lint a while back, but they supposedly fixed it.